ui/src/app/views/workflow/run/node/pipeline/service/service.log.component.ts

```diff
@@ -1,34 +1,38 @@
-import {Component, ElementRef, Input, NgZone, OnDestroy, OnInit, ViewChild} from '@angular/core';
+import { Component, ElementRef, Input, NgZone, OnDestroy, ViewChild } from '@angular/core';
 import * as AU from 'ansi_up';
-import {Subscription} from 'rxjs';
-import {environment} from '../../../../../../../environments/environment';
-import {ServiceLog} from '../../../../../../model/pipeline.model';
-import {PipelineStatus} from '../../../../../../model/pipeline.model';
-import {Project} from '../../../../../../model/project.model';
-import {WorkflowNodeJobRun, WorkflowNodeRun} from '../../../../../../model/workflow.run.model';
-import {AuthentificationStore} from '../../../../../../service/auth/authentification.store';
-import {AutoUnsubscribe} from '../../../../../../shared/decorator/autoUnsubscribe';
-import {CDSWebWorker} from '../../../../../../shared/worker/web.worker';
+import { Subscription } from 'rxjs';
+import { environment } from '../../../../../../../environments/environment';
+import { ServiceLog } from '../../../../../../model/pipeline.model';
+import { PipelineStatus } from '../../../../../../model/pipeline.model';
+import { Project } from '../../../../../../model/project.model';
+import { WorkflowNodeJobRun, WorkflowNodeRun } from '../../../../../../model/workflow.run.model';
+import { AuthentificationStore } from '../../../../../../service/auth/authentification.store';
+import { AutoUnsubscribe } from '../../../../../../shared/decorator/autoUnsubscribe';
+import { CDSWebWorker } from '../../../../../../shared/worker/web.worker';
 
 @Component({
     selector: 'app-workflow-service-log',
     templateUrl: './service.log.html',
     styleUrls: ['service.log.scss']
 })
 @AutoUnsubscribe()
-export class WorkflowServiceLogComponent implements OnInit, OnDestroy {
+export class WorkflowServiceLogComponent implements OnDestroy {
 
     @Input() project: Project;
     @Input() workflowName: string;
     @Input() nodeRun: WorkflowNodeRun;
     @Input('nodeJobRun')
     set nodeJobRun(data: WorkflowNodeJobRun) {
+        this.stopWorker();
         if (data) {
             this._nodeJobRun = data;
-            if (data.status === PipelineStatus.SUCCESS || data.status === PipelineStatus.FAIL || data.status === PipelineStatus.STOPPED) {
+            if (data.status === PipelineStatus.SUCCESS ||
```

Comment:
can you use `PipelineStatus.isDone(data.status)` instead please

Reply:
fixed